tests/pr_test.go

```diff
@@ -29,18 +29,12 @@ const solutionAgentsDADir = "solutions/agents"
 const agentsKubeconfigDir = "solutions/agents/kubeconfig"
 
 // Current supported regions for Observability instances
```

Comment:
suggest to update this comment to say its only including regions that event notifications support